Objective-C/CBLConsoleLogger.m

```diff
@@ -79,4 +80,17 @@ - (void) initializeOSLogDomains {
     osLogDictionary[@(kCBLLogDomainListener)] = os_log_create("com.couchbase.lite.ios", "Listener");
     #endif
 }
+
++ (os_log_t) internalLogger {
+    if(!logger){
+        logger = os_log_create("com.couchbase.lite.ios", "Log");
```

Comment:
This is not thread safe. You can use `dispatch_once` which is thread-safe to make it singleton.